### ISSUE TITLE
Lights in 3D scenes (gltf/glb) are imported as Spatial

### ISSUE THREAD

**Issue by sygi** [godotengine/godot#54941]
### Godot version

3.1.1-stable, v3.4.stable.official.206ba70f4

### System information

Linux Debian 5.10.46-5rodete1

### Issue description

The directional (and potentially other) lights from external 3D files are imported as (two) Spatial node. It should create a single light node.

### Steps to reproduce

Create a scene like this is blender:
![scene](https://i.ibb.co/3RgKLGs/Screenshot-from-2021-11-13-08-33-49.png)

Export it as glb/gltf:
![export settings](https://i.ibb.co/L9c1Vdy/Screenshot-from-2021-11-13-08-34-52.png)

Importing in godot results in:
![result](https://i.ibb.co/kSLKGp3/Screenshot-from-2021-11-13-08-36-09.png)

### Minimal reproduction project

Described above

**Comment by akien-mga** [comment]
Godot 3.1.1 was released 2.5 years ago and is no longer supported. Please test in Godot 3.4 or later.

**Comment by sygi** [comment]
The same problem exists in the newest version, updated the first message.

**Comment by Calinou** [comment]
https://github.com/godotengine/godot/issues/54596 claims that glTF lights are imported, albeit with incorrect energy when physical light attenuation is enabled.

**Comment by sygi** [comment]
It claims that "punctual" lights are imported with wrong energy. I claim directional are imported as Spatials. Do you know any free place to host a gltf file with my scene so that everyone can check?

**Comment by sygi** [comment]
Scratch that, it looks the file they provided is imported correctly and the one I did isn't. I looked into the files, and the nodes look identical:
```      
{
            "extensions" : {
                "KHR_lights_punctual" : {
                    "light" : 0
                }
            },
            "name" : "Light_Orientation",
            "rotation" : [
                -0.7071067690849304,
                0,
                0,
                0.7071067690849304
            ]
        },
        {
            "children" : [
                1
            ],
            "name" : "Light",
            "rotation" : [
                0.009320177137851715,
                0.7593702077865601,
                -0.4881025552749634,
                0.4301464259624481
            ],
            "translation" : [
                2.6132967472076416,
                3.3512892723083496,
                -3.4934568405151367
            ]
        },
```

**Comment by sygi** [comment]
After a more careful investigation I can confirm that this issue was indeed fixed between 3.1.1 and 3.4. I initially checked 3.4 by re-opening the same project, which had the .import files already generated by the 3.1.1 version. It looks the problem was there. After deleting the .import files and then reopening the project in 3.4, the lights are imported correctly.

Thanks for taking a look and sorry for the noise!
